deploy/cloud/modules/serverless/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "ec_url" {
+  description = "Elastic Cloud URL"
+  type        = string
+}
+
+variable "ec_apikey" {
+  description = "Elastic Cloud API Key"
+  type        = string
+}
+
+variable "project_name" {
+  description = "Elastic Cloud Project Name"
+  type        = string
+}
+
+variable "region_id" {
```

Comment:
Can we have a default region?

Reply:
We can add a default region here but it won't really help because the workflow environment variable will always override that value. Instead, I can change the default workflow env var to `aws-us-east-1`. What do you think?

Reply:
Now I see that ESS deployments don't support `aws-us-east-1` - we will have to find a region that is supported for both.